discovery-infra/test_infra/utils/terraform_util.py

```diff
@@ -0,0 +1,26 @@
+import logging
+import os
+
+from test_infra.consts import consts
+from test_infra.utils import utils
+
+
+class TerraformControllerUtil:
```

Comment:
maybe just delete this class and put all those methods in the module?
the module itself is named ``terraform_util``